docs/reference/config/browsers.mdx

```diff
@@ -1076,6 +1077,57 @@ The section has the following parameters:
     the application's backend source code.
 </Admonition>
 
+### stateOpts {#state_opts}
```

Comment:
This section 100% should not be inside "Selectivity" as it's not related to it. It should be either inside "running tests" or in a section of its own